frontend/src/components/MeetingRoom/CaptionsButton/CaptionsBox/CaptionsBox.tsx

```diff
@@ -0,0 +1,57 @@
+import { ReactElement } from 'react';
+import { Box } from '@mui/material';
+import UserCaption from './UserCaption';
+import useSessionContext from '../../../../hooks/useSessionContext';
+import useIsSmallViewport from '../../../../hooks/useIsSmallViewport';
+
+/**
+ * CaptionsBox Component
+ *
+ * This component shows a list of the captions that are currently in the meeting room.
+ * @returns {ReactElement} The captions box component.
+ */
+const CaptionsBox = (): ReactElement => {
+  const { subscriberWrappers, ownCaptions } = useSessionContext();
+  const isSmallViewPort = useIsSmallViewport();
+
+  const sxBox = {
+    position: 'absolute',
+    bottom: isSmallViewPort ? 100 : 80,
+    left: '50%',
+    transform: 'translateX(-50%)',
+    backgroundColor: 'rgba(0, 0, 0, 0.7)',
+    color: 'white',
+    px: 2,
+    py: isSmallViewPort ? 1 : 1.5,
+    borderRadius: 2,
+    width: isSmallViewPort ? '90vw' : 600,
+    height: isSmallViewPort ? 150 : 200,
+    overflowY: 'auto',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
+  };
+
+  return (
+    <Box data-testid="captions-box" sx={sxBox}>
+      {ownCaptions && (
```

Comment:
To confirm, we always want to display the publisher's captions first, yes?  If so, what happens if someone asks us a question and we respond.  Does our answer get rendered before their question?

Reply:
In your scenario, if person A asks a question while person B waits patiently and then responds, persons A captions will be rendered first but as person B answers it, assuming 4 seconds have not passed yet, their response might be rendered on top of person's A captions. We will have to display these in some order. If you prefer for the self-captions to go after other people's captions, I'll be okay to change that

Reply:
Naively, captions should be rendered in the order they're spoken.

In any case, let's see what feedback our users give us.  Maybe I'm splicing hairs.

Reply:
I agree with Kongpheng here, it makes more sense to render captions based on when someone is speaking or when the start of the caption is first received.

Reply:
I agree with you both. This wasn't a blocker for Kongpheng in this PR, though. Is it for you, @cpettet? I think it might get a bit tricky tracking the order and such.

Reply:
We've talked about re-reviewing Captions with Daniel once he's back from medical leave. I'll create one for that task in particular and add this as one of the items to review :pray: